docs/getting_started/running.md

```diff
@@ -1,63 +1,116 @@
+
+### Server PID File
+
+By default, the server writes its process ID (PID) to `/var/run/irods/irods-server.pid`.
+
+Use the `--pid-file` option to specify a different location. This is useful when running multiple iRODS servers on the same host.
+
+## Stopping the Server
 
-This mode instructs the server and all of its children to write log messages to **syslog**.
+iRODS supports two modes of shutdown.
 
-syslog can be configured to write messages locally, remotely, or both.
+### Fast Shutdown
+
+A fast shutdown is triggered by sending a `SIGTERM` or `SIGINT` to the main server process. For example:
 
 ```bash
-$ ./irodsctl start
+kill -TERM $(cat /var/run/irods/irods-server.pid)
 ```
 
-Messages can be rotated on a regular basis, most commonly through the **logrotate** service.
+This signal instructs agents to complete their current client request and terminate immediately (and cleanly), ending communication with the client.
+
+### Graceful Shutdown
 
-## Standard Output Mode
-This mode instructs the server and all of its children to write log messages to **stdout** instead of **syslog**. This mode can be enabled in two ways.
+A graceful shutdown is triggered by sending a `SIGQUIT` to the main server process. For example:
 
-### Option 1 (recommended) - Use the control script in the iRODS service account
 ```bash
-$ ./irodsctl start --stdout
+kill -QUIT $(cat /var/run/irods/irods-server.pid)
 ```
 
-### Option 2 - Launch the server binary directly
+This signal instructs agents to wait a limited amount of time before entering the [fast shutdown](#fast-shutdown) phase. During a graceful shutdown, agents will continue servicing client requests until the timeout. The timeout can be modified via the `graceful_shutdown_timeout_in_seconds` configuration property in `server_config.json`. See the section about [configuration](../../system_overview/configuration/#etcirodsserver_configjson) to learn more.
+
+## Reloading Server Configuration
+
+Unlike iRODS 4, iRODS 5 does not automatically load changes to `server_config.json`. Reloading the server's configuration requires sending a `SIGHUP` to the server. For example:
+
 ```bash
-$ /usr/sbin/irodsServer -u
+kill -HUP $(cat /var/run/irods/irods-server.pid)
 ```
 
-Regardless of the method, it is important to remember that once the server has been launched in this mode, the terminal will not return control until the server is shutdown or killed.
+On receiving `SIGHUP`, the server:
+
+1. Validates `server_config.json` against its schema.
+2. If validation **fails**, the reload is **rejected** and the previous configuration is used.
+3. If validation **succeeds**, the server:
+
+    - Updates its in-memory configuration
+    - Sends `SIGTERM` to the delay server (if running)
+    - Sends `SIGQUIT` to the agent factory
+    - Waits for the agent factory to close its listening socket
+    - Launches a new agent factory
```

Comment:
Does the agent factory spawn the delay server now? Regardless, maybe we should explicitly mention that a new delay server is launched.

Reply:
No. The main server process still launches the delay server.

Will include a statement about the delay server.